src/Signin.tsx

```diff
@@ -31,11 +32,13 @@ import {
 } from 'src/shared/constants'
 
 // Types
-import {RemoteDataState} from 'src/types'
+import {AppState, RemoteDataState} from 'src/types'
+import {getOrg} from 'src/organizations/selectors'
 
 interface State {
   loading: RemoteDataState
   auth: boolean
+  rudderStackInitialized: boolean
```

Comment:
instead of this state management, include the initialization in the cloud module, and not as a react component. then it only gets initialized once, when the first piece of code that requires it is included

Reply:
Done